app/src/main/java/org/mozilla/fenix/customtabs/ExternalAppBrowserFragment.kt

```diff
@@ -84,8 +84,8 @@ class ExternalAppBrowserFragment : BaseBrowserFragment(), UserInteractionHandler
                         activity,
                         components.core.store,
                         customTabSessionId
-                    ) { exception ->
-                        components.analytics.crashReporter.submitCaughtException(exception)
+                    ) { uri ->
+                        components.analytics.crashReporter.submitCaughtException(Exception("Unknown scheme error $uri"))
```

Comment:
@rocketsroger is this OK for fixing the breaking change?

Reply:
Sure, I'll apply my patch after

Reply:
👍